### ISSUE TITLE
Wayland support

### ISSUE THREAD

**Issue by arrufat** [lablup/backend.ai-webui#1625]
### What Operating System(s) are you seeing this problem on?

Linux (x86-64)

### Backend.AI version

22.09.14

### Describe the bug

Hi, I am running this app on Wayland, and I wish it had better support.

I am able to run it, but I don't get any window decorations on GNOME.

Command line, which works fine:
```
./Backend.AI\ Desktop --enable-features=UseOzonePlatform --ozone-platform=wayland
```

### To Reproduce

In order to get window decorations in GNOME Wayland, I should be able to run this:

```
./Backend.AI\ Desktop --ozone-platform-hint=auto --enable-features=WaylandWindowDecorations
```

This works when using Visual Studio Code, but here it gives:

```
(node:71407) ProtocolDeprecateCallback: The callback argument of protocol module APIs is no longer needed.
(Use `Backend.AI Desktop --trace-warnings ...` to show where the warning was created)
[71441:0309/102332.532448:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
[71407:0309/102334.965551:ERROR:browser_main_loop.cc(269)] Gdk: gdk_wayland_display_get_wl_display: assertion 'GDK_IS_WAYLAND_DISPLAY (display)' failed
fish: Job 1, './Backend.AI\ Desktop --ozone-p…' terminated by signal SIGSEGV (Address boundary error)
```

I've also tryed running:
```
./Backend.AI\ Desktop --enable-features=UseOzonePlatform --ozone-platform=wayland --enable-features=WaylandWindowDecorations
```

But I get the same error.

### Expected Behavior

I would expect the app not to crash and to have window decorations on GNOME Wayland

### Anything else?

I am guessing this could be resolved by updating the underlying electron version.

Also, I would appreciate it if I could have a file where I could specify the flags, like in Chromium (`~/.config/chromium-flags.conf`) or Visual Studio Code (`~/.config/code-flags.conf`)

**Comment by inureyes** [comment]
Related issue: https://github.com/lablup/backend.ai-webui/pull/1291

Since recent electron version has CORS / CORB breaking changes and technically App is an local-resource based WebApp, login features does not work after Electron 15. Therefore we added non-cookie based login method for electron-based app. However the app window authentication should be addressed before merging the branch above.

**Comment by inureyes** [comment]
@arrufat Now it is resolved via #1291
